### PR TITLE
Some refactoring and fixes

### PR DESCRIPTION
**Pull request by mitzkia**

### CODE REVIEW

File: Dockerfiles/basic-build-environments/debian_jessie/checkout_and_build_syslog_ng_incubator.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  cd /syslog-ng-incubator-src
+  git pull
+  autoreconf -i
+  mkdir build_dir && cd build_dir && mkdir -p ${INSTALL_DIR}
+  ../configure \
+    --prefix=${INSTALL_DIR}
+  make
+  make install
+else
+  echo "syslog-ng OSE incubator not supported with this OSE version: ${BRANCH_NAME}, it will be not downloaded, installed."
```
Comment (ihrwein) [inline Dockerfiles/basic-build-environments/debian_jessie/checkout_and_build_syslog_ng_incubator.sh]: What should happen when the `BRANCH_NAME` is not `master`?

Comment (mitzkia) [inline Dockerfiles/basic-build-environments/debian_jessie/checkout_and_build_syslog_ng_incubator.sh]: In this case I only want to accept certain branch for downloading and installing incubator (now only master branch)
This part can be changed if we want to support incubator with e.g OSE 3.6 and OSE 3.7.

Comment (ihrwein) [inline Dockerfiles/basic-build-environments/debian_jessie/checkout_and_build_syslog_ng_incubator.sh]: That's ok. I don't know what should be the result of the build. We don't support other branches, so we may  execute `false` to stop the building immediately. We may build "empty" images without a failing command, or I'm missing something.

Comment (ihrwein) [inline Dockerfiles/basic-build-environments/debian_jessie/checkout_and_build_syslog_ng_incubator.sh]: There was a little misunderstanding here on my side. We talked about this IRL and it's ok for me as it is.

**Comment by ihrwein** [comment]
@mitzkia 

review notes:
- create branch with descriptive name
- fix comments

**Comment by mitzkia** [comment]
Created a new pull request #9, this pull request is outdated. We can delete/close it.

**Comment by lbudai** [comment]
hi,

I think we should use the same commit message format rule as in syslog-ng, which is:

```
module: (brief) subject, containing  ~60 characters

description

<xxx-by section...>
<signed-off-by ...>
```

( For refactors : sometimes it is good to start with 'reasons/motivation' and only after the 'results'. )

**Pull request closed without merging**